docs/guide/connecting.asciidoc

```diff
@@ -15,22 +15,24 @@ page after you've created a cluster (look in the top-left if you're in Kibana).
 
 We recommend using a Cloud ID whenever possible because your client will be 
 automatically configured for optimal use with Elastic Cloud including HTTPS and 
-HTTP compression.
+HTTP compression; as well as the API key rather than
+the alternative `basic_auth=(user, password)` option.
 
 [source,python]
 ----
 from elasticsearch import Elasticsearch
 
-# Password for the 'elastic' user generated by Elasticsearch
-ELASTIC_PASSWORD = "<password>"
-
 # Found in the 'Manage Deployment' page
 CLOUD_ID = "deployment-name:dXMtZWFzdDQuZ2Nw..."
+# Generated under "Stack Management/Security" page
+API_KEY = "..."
 
 # Create the client instance
 client = Elasticsearch(
+    # For local development
+    # hosts=["http://localhost:9200"] 
```

Comment:
This is slightly confusing. Maybe it should be another section? Indeed, locally, using API Keys is more difficult - using the provided is much simpler. (Also, by default the URL is `https://` locally too).